src/main/java/org/broadinstitute/consent/http/db/mapper/DataAccessRequestMapper.java

```diff
@@ -31,21 +25,8 @@ public DataAccessRequest map(ResultSet resultSet, StatementContext statementCont
         dar.setSubmissionDate(resultSet.getTimestamp("submission_date"));
         dar.setUpdateDate(resultSet.getTimestamp("update_date"));
         String darDataString = resultSet.getObject("data", PGobject.class).getValue();
-        if (Objects.nonNull(darDataString)) {
-            // Handle nested quotes
-            String quoteFixedDataString = darDataString.replaceAll("\\\\\"", "'");
-            // Inserted json data ends up double-escaped via standard jdbi insert.
-            String escapedDataString = unescapeJava(quoteFixedDataString);
-            try {
-                DataAccessRequestData data = DataAccessRequestData.fromString(escapedDataString);
-                data.setReferenceId(dar.getReferenceId());
-                dar.setData(data);
-            } catch (JsonSyntaxException | NullPointerException e) {
-                String message = "Unable to parse Data Access Request, reference id: " + dar.getReferenceId() + "; error: " + e.getMessage();
-                logger.error(message);
-                throw new SQLException(message);
-            }
-        }
+        DataAccessRequestData data = translate(darDataString);
+        dar.setData(data);
```

Comment:
Not your code, but on line 19, we have a small bug. We need to check both for the column existence, and if the value is not null. `resultSet.getInt("collection_id")` returns 0 in the case of nulls.